### PR TITLE
KCL-6857 Move less files into Styles directory, remove css files, add…

### PR DESCRIPTION
**Pull request by jankalfus**
Move less files into Styles directory, remove css files, add compilation for less files

### Motivation

There was no automated compilation for less files before, and the resulting site.css was commited into the repository, which imposed problems (namely, styles from Admin.less were conflicting with the ones in Site.less and broke the Dancing Goat site, whenever they were rebuilt). The issues are fixed and the less files should build automatically.

### CODE REVIEW

File: DancingGoat/Styles/.gitignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+*.css
+*.css.map
```
Comment (PetrSvirak) [inline DancingGoat/Styles/.gitignore:2]: I think it should be in the root `.gitignore` (mentioned in https://github.com/Kentico/kontent-sample-app-net/pull/123#discussion_r639457467 already)

File: DancingGoat/wwwroot/css/.gitignore [hunk 2]
```diff
@@ -0,0 +1,2 @@
+﻿*.css
```
Comment (PetrSvirak) [inline DancingGoat/wwwroot/css/.gitignore:1]: This copy of the file is another reason for me to use the root `.gitignore` O:)

**Review by PetrSvirak — CHANGES_REQUESTED** [review]
While the introduction of `npm` is a bit questionable to me, I believe you mentioned that you found no other universal enough way to generate CSS files from LESS (e.g. that would not be tied to a single IDE). 

The change makes sense and not committing the built version of LESS files seems to overweight the cons. But that is just my opinion.

Anyway, the only vice I see is the extra `.gitignore` files

**Review by PetrSvirak — APPROVED** [review]

**Review by PetrSvirak — CHANGES_REQUESTED** [review]
The `npm ci` still failing during the automated build…

**Comment by jankalfus** [comment]
Closing in favor of https://github.com/Kentico/kontent-sample-app-net/pull/126

**Pull request closed without merging**